Cartesian_kernel/include/CGAL/Cartesian/Circle_2.h

```diff
@@ -69,6 +70,25 @@ class CircleC2
 
 };
 
+template < class R >
+typename R::Boolean
+CircleC2<R>::operator==(const CircleC2<R> &t) const
+{
+  if (CGAL::identical(base, t.base))
```

Comment:
That name `identical` is really a bad one.

Without any previous knowledge, how can one understand the difference between "equal" and "identical".

Probably `identical` should be renamed `are_same_objects`

Reply:
Adding this missing operator() was already barely in the scope of this PR and it's just a clone of the other ones.

There's a working group for the dev meeting on kernel work, we can add this there.